### ISSUE TITLE
Import Google Analytics data into Matomo

### ISSUE THREAD

**Issue by marisastrong** [CDLUC3/dmptool#516]
For a Google Analytics import into Matomo we need to

- [x] @marisastrong provide Joel access to the GA property, or you can just give him its Account ID, Property ID and View ID, which are listed in the GA property.
DMPTool Prd -GA4 is a duplicate of DMPTool Prd.  DMPTool Prd has more data.

- [x] Request 1 year prior to GA no longer being available
- [x] work with Eric/Joel to obtain access to the new Matomo property once it's available.

**Comment by briri** [comment]
Are we still working on this or did we decide that we don't need the old Google Analytics data?

**Comment by marisastrong** [comment]
This GA data import is underway into Matomo.  Once it is complete we'll be able to view it in Matomo.  I checked with Joel on status and will report back.

**Comment by mariapraetzellis** [comment]
Confirming we can view data back to July 30 2023
